orcid-web/src/main/java/org/orcid/frontend/web/controllers/AffiliationsController.java

```diff
@@ -298,6 +298,14 @@ private void editAffiliation(AffiliationForm affiliationForm) throws Exception {
             for (Affiliation affiliation : affiliationsList) {
                 AffiliationForm form = AffiliationForm.valueOf(affiliation);
                 form.setCountryForDisplay(getMessage(buildInternationalizationKey(CountryIsoEntity.class, form.getCountry().getValue())));
+                if(form.getDisambiguatedAffiliationSourceId() != null){
```

Comment:
Hey @lizkrznarich, just one comment here, are we sure that if **form.getDisambiguatedAffiliationSourceId()** is not null, then **form.getOrgDisambiguatedId()** will not be null either? why not changing the if condition to check for the presence of **form.getOrgDisambiguatedId()** instead?